user_manual/ios/ios_app.rst

```diff
@@ -0,0 +1,93 @@
+==========================
+Using the ownCloud iOS App
+==========================
+
+Accessing your files on your ownCloud server via the Web interface is easy and 
+convenient, as you can use any Web browser on any operating system without 
+installing special client software. However, the ownCloud iOS app offers 
+some advantages over the Web interface:
+
+* A simplified interface that fits nicely on an iPhone or iPad
+* Automatic synchronization of your files
+* Easily upload files from your device to ownCloud
+* Two-factor authentication
```

Comment:
I'd call this "PIN Protection" or so – because effectively there is only one factor, the other one is stored and you're not asked at all.